.gitignore

```diff
@@ -73,3 +73,6 @@ doc/*
 
 service_account.json
 /.cache
+
+# asdf
+.tool-versions
```

Comment:
For those using asdf like myself, and setting `asdf local ruby 3.3.5` sets a .tool-versions file. I'm adding it to the `.gitignore` file. Let me know if you think otherwise.